### DISCUSSION TITLE
Functions for the interface.

### DISCUSSION THREAD

**Discussion by michail935** [deforum-art/sd-webui-text2video#85] [Ideas]
Greetings to the developers. Thank you for your work. There is a suggestion. The interface has a line "seed" and as I understand it, you can add a seed from any already generated image. It would be nice if you could add a "seed" button. So that you can make changes from an already generated video. That is, seed from the video. Button to add and reset the "seed"

**Comment by aivrar** [comment]
Yes, if we could get that seed for later usage that would be awesome. Also, would be helpful if we could get a text file in the directory of all the settings that were used for later reference. Great extension I love it!
